docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc

```diff
@@ -25,12 +30,29 @@ If you want to create an application using source code from GitLab, you need to
 . Optional: Under **Labels**, add a label to tag or provide more context for your secret.
 . Click **Add secret**.
 
-== Creating secrets for GitLab-sourced apps
+== Creating registry pull secrets
 
-Before you create an application using source code from GitLab, generate an access token for your GitLab repository and upload it to your {ProductName} workspace.
+Some container builds may use parent images from registries that require authentication (i.e. `registry.redhat.io`). Until these credentials have been configured, the builds will continue to fail due to the system being unable to pull the required images.
```

Comment:
```suggestion
Some container builds may use parent images from registries that require authentication, for example, `registry.redhat.io`. Until these credentials have been configured, the builds will continue to fail due to the system being unable to pull the required images.
```
As per the IBM style guide, we should avoid using i.e. , e.g. and any other similar conjunctions.